exonum-java-binding/core/src/main/java/com/exonum/binding/core/transaction/ExecutionException.java

```diff
@@ -18,28 +18,43 @@
 
 import com.exonum.binding.common.hash.HashCode;
 import com.exonum.binding.core.blockchain.Blockchain;
+import com.exonum.core.messages.Runtime.ErrorKind;
+import com.exonum.core.messages.Runtime.ExecutionError;
 import com.exonum.core.messages.Runtime.ExecutionStatus;
 import javax.annotation.Nullable;
 
 /**
- * An exception occurred during transaction execution. The transaction exception includes
- * an integer error code, that may be either service-specific or transaction-specific;
- * and an optional description — an exception message. Both the error code and the description
- * are saved in the database, but only the value of the error code affects the blockchain state.
+ * An error occurred during the execution of a Service method. The execution exception includes
+ * an integer error code, that may be either service-specific or operation-specific;
```

Comment:
actually it's byte, not integer.
Are there any restrictions for this value i.e. `[0, 127]`?

Reply:
byte is an integer (or integral) type. It doesn't say "`int`".